contracts/test/auth.ts

```diff
@@ -149,20 +154,14 @@ describe('Auth', function () {
     await expect(siweAuthTests.testVerySecretMessage(bearer3)).to.be.reverted;
 
     // Expired bearer.
-    const siweStr4 = await siweMsg('localhost', 0, new Date(Date.now() + 10)); // Expire after 0.01 seconds.
+    const expiration = new Date(Date.now() + 500);
+    const siweStr4 = await siweMsg('localhost', 0, expiration); // Expire after 0.5 seconds.
     const bearer4 = await siweAuthTests.testLogin(
       siweStr4,
       await erc191sign(siweStr4, account),
     );
-    // Wait for 2 blocks. The first block may still have a timestamp earlier than the expiration time.
-    await new Promise(async (r) => {
-      const desiredBlockNumber = (await ethers.provider.getBlockNumber()) + 2;
-      ethers.provider.on('block', (blockNumber) => {
-        if (blockNumber == desiredBlockNumber) {
-          r();
-        }
-      });
-    });
+    await delay(expiration.getTime() - new Date(Date.now()).getTime());
```

Comment:
`new Date(Date.now()).getTime()` looks like a typo but works. It's equal to `new Date().getTime()` and `Date.now()` tho

Reply:
Yup it's equivalent. I think I had a minor brainfart.